changelogs/unreleased/gh-7746-fiber-channel-graceful-close.md

```diff
@@ -0,0 +1,5 @@
+## feature/fiber
```

Comment:
> NO_DOC=docs on tarantool.wiki

There is [API description](https://www.tarantool.io/en/doc/latest/reference/reference_lua/fiber/#channels) on the website and it seems it should be updated after the patch. We should add a documentation request. Let's mention existing documentation inside.

Reply:
Done:                                
>  Title: display availible compat options in doc                                                                                                                                                                
  With introduction of tarantool.compat, there will be added a lot of                               
  options that change user-visible behavior. Such changes are described in                          
  detail on tarantool.wiki, but it could be benefitial to have a link to                            
  them in the documentation of the current behavior.

Reply:
Hm. This task is not measurable: it has no certain end, because the options are added from time to time. It'll be not clear, when one can close the task. I would rephrase it against the option introduced here.

Reply:
Updated:
```
    @TarantoolBot document
    Title: new compat option fiber_channel_graceful_close
    
    New behavior is gracefully closing fiber channel by marking it
    read-only, instead of destroying. Full API description can be found
    on notion (private):
    https://www.notion.so/tarantool/Fiber-channel-graceful-close-53b2788ed1f144598c4c0e1229c2eb69
```

Reply:
Thank you!